api/internal/humiographql/scheduled-search.go

```diff
@@ -0,0 +1,57 @@
+package humiographql
+
+import graphql "github.com/cli/shurcooL-graphql"
+
+type ScheduledSearch struct {
+	ID                         graphql.String   `json:"id"`
+	Name                       graphql.String   `json:"name"`
+	Description                graphql.String   `json:"description,omitempty"`
+	QueryString                graphql.String   `json:"queryString"`
+	Start                      graphql.String   `json:"start"`
+	End                        graphql.String   `json:"end"`
+	TimeZone                   graphql.String   `json:"timezone"`
+	Schedule                   graphql.String   `json:"schedule"`
+	BackfillLimit              graphql.Int      `json:"backfillLimit"`
+	Enabled                    graphql.Boolean  `json:"enabled"`
+	ActionsV2                  []Action         `json:"actionsV2"`
+	RunAsUser                  User             `json:"runAsUser,omitempty"`
+	LastScheduledSearch        Long             `json:"lastScheduledSearch"`
+	TimeOfNextPlannedExecution Long             `json:"timeOfNextPlannedExecution"`
```

Comment:
These are execution info and as thus I do not think that they should be part of the CLI, we also did not add them for filter and aggregate alerts.

Reply:
I'll remove them